packages/shared/lib/services/connection.service.ts

```diff
@@ -2049,6 +2049,24 @@ class ConnectionService {
             return { success, error, response: success ? (creds as OAuth2Credentials) : null };
         }
     }
+
+    // return the number of active connections per account
+    async countMetric(): Promise<Result<{ accountId: number; count: number }[], NangoError>> {
+        const res = await db.knex
+            .withSchema('nango')
+            .from('_nango_connections')
+            .debug(true)
+            .join('_nango_environments', '_nango_connections.environment_id', '_nango_environments.id')
+            .count('_nango_connections.id as count')
+            .select<{ accountId: number; count: number }[]>('_nango_environments.account_id')
+            .where('_nango_connections.deleted_at', null)
+            .groupBy('_nango_environments.account_id');
```

Comment:
Query is currently <500ms in prod. The query plan could be improved but I am cautious about adding indexes for a single query that runs once every 5mins